example/coqdocjs

```diff
@@ -0,0 +1 @@
+..
```

Comment:
This introduces a cycle that make complains about. Please create a `coqdocjs` folder and only link to `extra` and `Makefile.doc`